src/jquery.uls.core.js

```diff
@@ -348,11 +358,11 @@
 
 			languagesCount = objectLength( this.options.languages );
 
-			if ( languagesCount < 12 ) {
+			if ( languagesCount < 25 ) {
 				return 'narrow';
 			}
 
-			if ( languagesCount < 100 ) {
+			if ( languagesCount < 400 ) {
```

Comment:
I have a comment about this at https://phabricator.wikimedia.org/T138235#2581284